src/components/common/Rating/RatingComponent.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import Rating from 'react-rating';
+import style from './RatingComponent.scss';
+
+const theme = {
+  color: '#ffd700',
+  border: '1px solid #fff',
+  fontSize: '13px',
+};
+export default class RatingArticleCardComponent extends Component {
+  render() {
+    return (
+      <div className="landingArticleCardRating" style={style}>
+        <div className="landingArticleCardRating--rating">
+          <p>Rating:</p>
+        </div>
+        <div className="landingArticleCardRating--stars">
+          <Rating
+            style={theme}
+            emptySymbol="far fa-star"
+            fullSymbol="fas fa-star"
+            readonly
+          />
+        </div>
```

Comment:
we already have a rating component

Reply:
I am now using the common rating component thanks for the feedback